src/components/LandingPage/Categories/Categories.jsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom"
+import "./Categories.css"
+export default function Categories(){
+    return (
+        <section className="categories-base">
+        <div className="grids-5">
+            <div className="grid-child">
+                <img src={process.env.PUBLIC_URL + 'assets/Categories_Images/planners.webp'} alt="Category-Planners"/>
+                <div className="overlay-text grey-badge"><Link className="link-text" to="/products">Planners</Link> </div>
+            </div>
+            <div className="grid-child">
+                <img src={process.env.PUBLIC_URL + 'assets/Categories_Images/journals.jpg'} alt="Category-Journals"/>
+                <div className="overlay-text grey-badge"><Link className="link-text" to="/products">Journals</Link></div>
+            </div>
+            <div className="grid-child">
+                <img src={process.env.PUBLIC_URL + 'assets/Categories_Images/washiTapes.webp'}
+                    alt="Category-WashiTapes"/>
+                <div className="overlay-text grey-badge"><Link className="link-text" to="/products">Washi Tapes</Link></div>
+            </div>
+            <div className="grid-child">
+                <img src={process.env.PUBLIC_URL + 'assets/Categories_Images/pens.jpg'} alt="Category-Pens"/>
+                <div className="overlay-text grey-badge"><Link className="link-text" to="/products">Pens</Link></div>
+            </div>
+            <div className="grid-child">
+                <img src={process.env.PUBLIC_URL + 'assets/Categories_Images/stamps.webp'} alt="Category-Stamps"/>
+                <div className="overlay-text grey-badge"><Link className="link-text" to="/products">Stamps</Link></div>
+            </div>
+        </div>
```

Comment:
Suggestion: You could create an array having category object as item and them could map over it to render on view.

Reply:
I agree. This would make it much cleaner. Also, I assume that you will be using the mockbee category db, so this approach wont be feasible then.

Reply:
Ohh! Right, thank you! This skipped my mind.